src/server.c

```diff
@@ -3672,6 +3679,18 @@ void slowlogPushCurrentCommand(client *c, struct redisCommand *cmd, ustime_t dur
     slowlogPushEntryIfNeeded(c,argv,argc,duration);
 }
 
+/* This function is called in order to update the total command histogram duration.
+ * If needed it will allocate the histogram memory and trim the duration to the upper/lower tracking limits*/
+void updateCommandLatencyHistogram(struct hdr_histogram** latency_histogram, int64_t duration_hist){
```

Comment:
We stick pointers with the variable
```suggestion
void updateCommandLatencyHistogram(struct hdr_histogram **latency_histogram, int64_t duration_hist){
```